scenarios/04_Battle_of_Barnon.cfg

```diff
@@ -789,7 +888,85 @@
         [filter_second]
             side=1
         [/filter_second]
-        {MSG_narrator _"How did you manage to kill Bor Cryne? This shouldn't happen... it means the level is too easy. Mail adanilecki @ cs put poznan pl with your savegame and replay for review!"}
+#ifdef DEBUG_MODE
+        #po: Bor Cryne shouldn't die either way, whether debug mode is on or not, so the fact that we recognize debug mode specially here is just an easter egg:
+        {MSG_narrator _"Using debug mode to kill Bor Cryne is cheating!"}
```

Comment:
I don't like ending the level just because debug mode is on. Maybe just give the player the meta comment without actually ending the scenario. It's kind of tacky to do it like this imo.

Reply:
ok, removed that part in 61ba872